distribution/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.12.1-slim-bullseye
+
```

```suggestion
ENV PYTHONPATH="${PYTHONPATH}:/src/:/src/product/"
```
чтобы питон смог найти наши внутренние скрипты, позволит избежать проблемы импорта